neuralprophet/time_net.py

```diff
@@ -163,8 +163,17 @@ def __init__(
         self.batch_size = self.config_train.batch_size
 
         # Metrics Config
+        METRICS = {
```

Comment:
We currently have `METRICS` defined in here and `utils_metrics.py`. We should only have one source for this dict. Can we consolidate this?

Reply:
only defined in utils_metrics.py